.github/workflows/ci.yml

```diff
@@ -188,6 +188,44 @@ jobs:
     - name: Check formatting
       run: cargo fmt -- --check
 
+  gather_workspace_members:
+    runs-on: ubuntu-latest
+    outputs:
+      members: ${{ steps.cargo-metadata.outputs.members }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: cargo-metadata
+        run: |
+          WORKSPACE_MEMBERS=$(cargo metadata --format-version=1 --no-deps | jq -c '.packages |.[] | .name' | jq -s | jq -c '.')
+          echo "::set-output name=members::${WORKSPACE_MEMBERS}"
+
+  semver-check:
+    runs-on: ubuntu-latest
+    needs: gather_workspace_members
+    strategy:
+      matrix:
+        crate: ${{ fromJSON(needs.gather_workspace_members.outputs.members) }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install rust nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
```

Comment:
Do we need to run on nightly? This will poison our cache effectively daily ...

Reply:
Good catch